backend/src/main/kotlin/app/ehrenamtskarte/backend/application/database/repos/ApplicationRepository.kt

```diff
@@ -158,10 +154,12 @@ object ApplicationRepository {
                 true
             }
         }
-    }
 
-    fun delete(applicationId: Int, graphQLContext: GraphQLContext): Boolean {
-        return transaction {
+    fun delete(
+        applicationId: Int,
+        graphQLContext: GraphQLContext,
+    ): Boolean =
```

Comment:
:wrench: Is there a way to configure this to not force the linebreaks for multiple parameters here? I think its easier to read without it.
```suggestion
    fun delete(applicationId: Int, graphQLContext: GraphQLContext): Boolean =
```